book/src/specification/textual-representation/concrete-syntax.md

```diff
@@ -17,22 +17,23 @@ process.
 
 ## Items
 
-### Alias definitions
+### Constant definitions
 
-Alias definitions are used to give names to terms that can be later used in
+Constant definitions are used to give names to terms that can be later used in
 other places in the binary description. For example:
 
 ```fathom
-Byte = U8;
+const Byte = U8;
```

Comment:
There are more examples than just this one in the book ;)

Reply:
Oops, i just grepped for "constant" or "alias" :stuck_out_tongue:

Reply:
I had a look at each Markdown file in the `book` directory, i think it's done now?

Reply:
Looks great!